src/nvim/macros.h

```diff
@@ -127,6 +127,13 @@
     (wp)->w_p_crb = false; \
   } while (0)
 
+
+// -V:ARRAY_SIZE:1063
```

Comment:
Can this be moved in the same comment block for `ARRAY_SIZE` macro, specifically the line before `#define`?

There's already an explanation for the 2nd division so modifying that explanation instead.

Reply:
Of course. I've updated the commit, did I interpret you correctly?